packages/usa-footer/src/index.mjs

```diff
@@ -88,10 +88,12 @@ module.exports = behavior(
       this.mediaQueryList = window.matchMedia(
         `(max-width: ${HIDE_MAX_WIDTH - 0.1}px)`
       );
+      // ! Deprecated feature
       this.mediaQueryList.addListener(resize);
     },
 
     teardown() {
+      // ! Deprecated feature
       this.mediaQueryList.removeListener(resize);
```

Comment:
**TODO: Deprecated feature**

The `mediaQueryList.addListener` and `removeListener` are deprecated features per [MDN](https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/addListener). Probably why the unit test `footer.spec.mjs` imports `mq-polyfill` because of this. Removing the polyfill causes the test to fail.